src/components/Settings/sections/GpuAccelerationNotice.scss

```
@@ -0,0 +1,57 @@
+    font-size: vars.$font-caption;
+    font-weight: 600;
+    color: vars.$color-warning;
+    line-height: 1.4;
+  }
+
+  &__text {
+    margin: 4px 0 0;
+    font-size: vars.$font-caption;
+    color: vars.$text-secondary;
+    line-height: 1.4;
+    word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)